requirements.txt

```diff
@@ -4,13 +4,11 @@ keyring
 keyrings.alt
 
 # GitPython 3.1.15 deprecates Python 3.5
-GitPython==3.1.14; python_version >= '3.0' and python_version < '3.6'
-GitPython; python_version >= '3.6' or python_version <= '3.0'
+GitPython==3.1.14; python_version < '3.6'
```

Comment:
No more need for this, since we'll only support Python 3.6+ ?

Reply:
f9309b8